File: internal/collector/treesitter/testdata/test_kind/typescript/js-playwright/test/login.spec.ts
// SPDX-License-Identifier: Apache-2.0

import { test, expect } from "@playwright/test";

test.describe("login", () => {
  test("succeeds", async ({ page }) => {
    await page.goto("/");
    await expect(page).toHaveTitle(/Home/);
  });
});

File: internal/collector/treesitter/testdata/test_kind/ruby/ruby-rspec/test/login_spec.rb
# SPDX-License-Identifier: Apache-2.0

RSpec.describe "login" do
  it "succeeds" do
    expect(1 + 1).to eq(2)
  end

  context "when password missing" do
    it "fails" do
      expect(true).to be true
    end
  end
end
